check-approved-requests.py

```diff
@@ -203,6 +214,17 @@ def check_requests(request_type, auth_file, worksheet_key):
             processed_rows.append(idx)
             if args.log:
                 log_request(args.log, timestamp, request_info['user_email'])
+
+        elif (row[0] == '') and (datetime.now() >= dateparser.parse(row[2]) +
+                                 timedelta(hours=24)):
+            # send reminder about rows that have been waiting for approval
+            # for more than 24 hours
+            request_info = parse_function(row)
+            reminder(template=reminder_template,
```

Comment:
`request_type` is already a variable in this check_requests function, pass it as an argument to `reminder()`